android/build.gradle

```diff
@@ -45,5 +45,5 @@ repositories {
 
 dependencies {
     api 'com.facebook.react:react-native:+'
-    api 'io.radar:sdk:3.5.10'
+    api 'io.radar:sdk:3.5.13'
```

Comment:
We should probably set this to `3.5.12` to keep it consistent with iOS (Also watching what's happening with Android, I think there's going to be another release in a day or two)

Reply:
Makes sense to me! Doing now